### DISCUSSION TITLE
Is there any way to align the text inside CTkButton to left or right?

### DISCUSSION THREAD

**Discussion by Zingzy** [TomSchimansky/CustomTkinter#1105] [Q&A] [answered]
Is there any way to align the text inside CTkButton to left or right?

**Comment by Akascape** [comment, marked as answer]
@Zingzy Yes, the text can be aligned with the anchor parameter.
Just add `anchor="w"` for left and `anchor="e"` for right.

**Reply by LucianoSaldivia** [reply]
Hey, i think i'm doing just that and it's not working... I just posted an issue [here](https://github.com/TomSchimansky/CustomTkinter/issues/1393)  with a minimal example, would you mind lending me a hand?
